metrics/interceptors.go

```diff
@@ -1,23 +1,30 @@
 package metrics
 
 import (
+	"bytes"
 	"context"
 	"strings"
 	"time"
 
 	"github.com/samber/lo"
+	"go.uber.org/atomic"
 	"google.golang.org/grpc"
 
 	"github.com/anyproto/anytype-heart/util/debug"
 	"github.com/anyproto/anytype-heart/util/reflection"
 )
 
 const (
-	unexpectedErrorCode      = -1
-	parsingErrorCode         = -2
-	defaultUnaryWarningAfter = time.Second * 3
+	unexpectedErrorCode = -1
+	parsingErrorCode    = -2
 )
 
+var (
+	// every duration will be added to the previous ones
+	UnaryTraceCollections = []time.Duration{time.Second * 3, time.Second * 7, time.Second * 10, time.Second * 30, time.Second * 50} // 3, 10, 20, 50, 100
```

Comment:
misleading comment
// 3, 10, 20, 50, 100
this is not true

Reply:
@fat-fellow because it runs after each other, the durations are cumulative